src/components/show-products/index.js

```diff
@@ -155,7 +155,8 @@ export const ShowProducts = () => {
                     id={id}
                     name={productName}
                     onChange={(event) => handleCheckboxChange(event, id)}
-                    checked={checked}
+                    checked={!!checked}
```

Comment:
Why they needed to change use `!!` on here?

Reply:
It's because we have to fix the behavior of the input to controlled component using this operator to convert  undefined into truthy value
https://reactjs.org/docs/forms.html#controlled-components

Reply:
Wow! This is amazing! Thanks for the docs! ❤️